cmd/juju/application/refresh.go

```diff
@@ -174,8 +174,8 @@ uploaded with the revision specified in the charm, if possible, otherwise it
 gets a unique revision (highest in state + 1).
 
 When deploying from a path, the --path option is used to specify the location from
-which to load the updated charm. Note that the directory containing the charm must
-match what was originally used to deploy the charm as a superficial check that the
+which to load the updated charm. Note that the charm must match what was 
+originally used to deploy the charm as a superficial check that the
```

Comment:
its not clear what form the charm is in here. Be specific about the location of the charm package. Or whichever term we use.

Reply:
There is a reference to local charm paths at line 167 as well.